src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorDocumentMappingService.cs

```diff
@@ -77,6 +78,14 @@ public override bool TryMapToProjectedDocumentRange(RazorCodeDocument codeDocume
                 return false;
             }
 
+            // Ensures a valid range is returned
```

Comment:
Instead of doing this(or in addition to doing this), is there any harm in rejecting if `originalRange.End < originalRange.Start`? Do we want to accept such an input?

Reply:
I can add it in, not sure if it may cause issues elsewhere.

Reply:
Yeah. Let's ensure the input range is valid. If this causes issues, then those are valid bugs we should address.

Reply:
Updated.